src/site/paragraphs/number_callout.drupal.liquid

```diff
@@ -0,0 +1,18 @@
+{% comment %}
+  Example data:
+    "entity": {
+        "entityBundle": "number_callout",
+        "entityId": "938",
+        "fieldShortPhraseWithANumber": "Less than 1 week",
+        "fieldWysiwyg": {
+            "processed": "..."
+        }
+    }
+{% endcomment %}
+
+<div itemprop="itemListElement">
```

Comment:
This `itemprop` attribute doesn't seem like something that should be in the number callout template. It's specific to the QA markup and the number callout could be used in other contexts.

Reply:
@jbalboni do i need the `markdown=0` attribute?

Reply:
No, you can remove that, too